unittests/rpc/handlers/AccountTxTests.cpp

```diff
@@ -464,8 +464,12 @@ TEST_F(RPCAccountTxHandlerTest, IndexSpecificForwardTrue)
             testing::_))
         .Times(1);
 
+    auto const rawETLPtr = dynamic_cast<MockETLService*>(mockETLServicePtr.get());
```

Comment:
`ASSERT_NE(rawETLPtr)`, but clang-tidy should catch it 🙂

Reply:
weird . it does not catch for me